src/targets/gpu/prefuse_ops.cpp

```diff
@@ -256,25 +256,9 @@ struct base_group_query_attention
     }
 };
 
-struct gpu_compute_attention_probabilities : base_group_query_attention
+struct gpu_kv_cache_attention : base_group_query_attention
```

Comment:
Instead of creating a new op like this, can you just add the `group` operator like in #3977.